packages/core/parcel/src/cli.js

```diff
@@ -152,17 +152,19 @@ function normalizeOptions(command): ParcelOptions {
   if (command.name() === 'serve') {
     serve = {
       https,
-      port: command.port,
-      host: command.host
+      port: command.port || 1234,
+      host: command.host,
+      certificateDir: '.parcel-cert'
```

Comment:
Would we not just put it in .parcel-cache?

Reply:
In Parcel 1 it's pretty frustrating that the self signed certificate changes everytime you clear the cache. But in Parcel 2 cache should be stable so not sure if this should be seperate?

Reply:
Yeah... well hopefully it should be better. I think it would be annoying to need to ignore two separate parcel dirs in `.gitignore`.